src/fparser/two/Fortran2003.py

```diff
@@ -6263,6 +6324,7 @@ def tofortran(self, tab='', isfix=None):
                 extra_tab += '  '
         if len(self.content) > 1:
             line.append(end.tofortran(tab=tab, isfix=isfix))
+        py2_encode_list_items(line)
```

Comment:
Add a test for this as was done for r438 and r808?

Reply:
Done.